code/game/objects/effects/spawners/minor_variation.dm

```diff
@@ -0,0 +1,58 @@
+/*
+Makes variations possible with code only.
+This is mainly to prevent creating new map files when creating smaller variations that aren't worth a map file. - Hopek
+*/
+
+/obj/effect/variation
+	name = "variation"
+	icon = 'icons/effects/landmarks_variation.dmi'
+	icon_state = "generic_green"
+	var/variations // number of variations
+	var/choice // chosen variation by choice()
+	var/chosen // has a choice been made for this place?
+	var/location_place // location of placement calculated from reference
```

Comment:
no need for this to be a var on the thing i think

Reply:
Is there a point?
The object gets deleted as soon as the spawning is done.

Reply:
See my comment below